### DISCUSSION TITLE
Some cave hazards could be more of a threat

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#11791] [Ideas and suggestions]
Mushroom cave bulbs do very little burn damage when they pop, which gets reduced even more since most diving suits have pretty decent burn resistance. Players can run into them fairly consequence free. Upping their damage and changing their burn to acidburn (seems more fitting) would hopefully make them more of a hazard.

Crystal cave shards are somewhat fine since lacerations don't heal by themselves and the bleeding might cause some bloodloss but I've personally never felt like I had to be extra careful around them. Perhaps they could also use a bump in damage as they still feel like a minor annoyance rather than a cave hazard.

**Comment by Rotiart-Drol** [comment]
I feel like cave hazards serve their purpose appropriately. They're a low-threat obstacle, they shouldn't be causing you much grief even if you do a poor job at avoiding them
